packages/components/src/Chip/Chip.css

```diff
@@ -24,7 +24,12 @@
   align-items: center;
 }
 
-.chip:focus {
+.chip:focus,
+.chip:focus-visible {
+  outline: none;
```

Comment:
General note for this PR: I was wondering if just applying `:focus-visible` styling is enough, but in testing at a [basic level](https://codepen.io/murhaus/pen/KKJyJjQ) it _looks like_ there needs to be something telling the element not to receive a focus ring in the `:focus` state even if `:focus-visible` is declared.

Reply:
![image](https://github.com/GetJobber/atlantis/assets/39704901/ef7830c0-9c78-4bf5-aa71-dca146fa3ea3)